test/check-machines-networks

```diff
@@ -122,7 +122,7 @@ class TestMachinesNetworks(machineslib.VirtualMachinesCase):
         # Transient network cannot be deleted
         b.wait_visible(f'li.pf-m-aria-disabled > #delete-network-test_network4-{connectionName}')
         b.click(f'#deactivate-network-test_network4-{connectionName}')  # Deactivate transient network
-        self.waitNetworkRow("test_network4", connectionName, False)  # Check it's not present after deactivation
+        self.waitNetworkRow("test_network4", connectionName, present=False)  # Check it's not present after deactivation
```

Comment:
This change should not be part of "Remove useless React fragment". Rebase error?

Reply:
Woops!